copier.yml

```diff
@@ -62,10 +69,10 @@ command_line_interface:
         - Argparse
         - No command-line interface
     help: Which command line interface do you want to use?
-create_author_file:
-    type: bool
-    default: true
-    help: Do you want to create an author file?
+#create_author_file:
+#    type: bool
+#    default: true
+#    help: Do you want to create an author file?
```

Comment:
Is it worth just deleting this entirely? It seems like the pattern is fairly obvious if you ever needed to add it back in

Reply:
Yes, I forgot this one. Thanks!